sdk/python/kfp/dsl/_volume_op.py

```diff
@@ -44,6 +44,7 @@ def __init__(self,
                  modes: List[str] = None,
                  annotations: Dict[str, str] = None,
                  data_source=None,
+                 action: str = "create",
```

Comment:
Can you add `action` to args documentation?

Reply:
Will resolve in the new revision.